metadata_service/config.py

```diff
@@ -113,6 +113,10 @@ class AtlasConfig(LocalConfig):
     # format to be used in tables
     WATERMARK_DATE_FORMATS = ['%Y%m%d']
 
+    # Qualified Name of the Glossary, that holds the user defined terms.
+    # For Amundsen, we are using Glossary Terms as the Tags.
+    ATLAS_USER_DEFINED_TERMS = 'USER_DEFINED_TERMS'
```

Comment:
if the name is amundsen specific enough then this doesn't have to be a configuration option. I would rather make it AtlasProxy attribute rather than config option as no other proxy will have it and I think it's not necessary to have this as config option.

Reply:
This is not really amundsen specfic. 
Let's say if someone is using glossary terms for multiple purposes, and want to use one of the existing glossary terms as user defined tags, then this is the place to do that. I belive making it configurable for now will make it easy for the future.

Reply:
What is the difference between user defined and non-user-defined tags ? Afaik only one is the user-defined are editable from UI, so it is Amundsen specific.

Reply:
If we're keeping this then I'd make default something Amundsen specific like `amundsen_user_tags`.

Reply:
Nope, it will go to AtlasProxy. I think it make nore sense if we move this out of config and make it a part of proxy class instead.